api/DataTransferItem.json

```diff
@@ -99,6 +99,8 @@
       },
       "getAsFileSystemHandle": {
         "__compat": {
+          "spec_url": "https://wicg.github.io/file-system-access/#dom-datatransferitem-getasfilesystemhandle",
```

Comment:
Looks like there's no lint for this, but can you put mdn_url before spec_url?

Reply:
Sure! Fix in next commit.